2_process/calc_doy_means.py

```diff
@@ -24,5 +23,7 @@ def main(out_file, in_file, lake_id):
     calc_doy_means(df, lake_id, f"2_process/out/doy_{lake_id}.csv")
```

Comment:
Should this line use `out_file` instead of hardcoding the output file path? I think Ted and I messed up here. I haven't tested this code, but maybe it should be:
```suggestion
    calc_doy_means(df, lake_id, out_file)
```
If so, we'll want to change it in the starter code, too.